### ISSUE TITLE
VGG acc always keeps 0

### ISSUE THREAD

**Issue by GeKeShi** [arashno/tensorflow_multigpu_imagenet#18]
I train vgg type D(default) from scratch, but the loss doesn't decline, and the acc always keeps 0, others model like resnet50 is fine. Is there anyone meet the same problem?

**Comment by arashno** [comment]
Yes, sometimes it happens for the VGG.
Have you tried lowering the learning rate?

**Comment by hiJulie** [comment]
Have you solved this problem? Can you give me some suggestions for solving this problem? I tried to reduce the learning rate and still invalid

**Comment by GeKeShi** [comment]
> Have you solved this problem? Can you give me some suggestions for solving this problem? I tried to reduce the learning rate and still invalid

no, I haven't, I replace it with alexnet and resnet, but the training speed is too slow to weeks
